spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -32,21 +32,29 @@ def and_i_upload_a_file_attachment
     @publishing_api_request = stub_publishing_api_put_content(@edition.content_id, {})
 
     @attachment_filename = "13kb-1-page-attachment.pdf"
```

Comment:
I noticed we have an instance variable here but don't use it outside of this method.  I think we could either use it in lines 43 and 44, or make it a local variable.

Reply:
Good point, I've used that variable on lines 43 and 44 now.